library/common/jni/BUILD

```diff
@@ -19,3 +22,18 @@ cc_binary(
     linkshared = True,
     deps = ["//library/common:envoy_main_interface_lib"],
 )
+
+local_jar_cc_binary(
+    name = "java_libenvoy_jni.so",
+    srcs = [
+        "java_utility.cc",
+        "jni_interface.cc",
+        "jni_utility.h",
+    ],
+    includes = [
+        "../../../external/local_jdk/include",
+        "../../../external/local_jdk/include/darwin",
+        "../../../external/local_jdk/include/linux",
```

Comment:
Will this relative path break if things move around? Can it be an absolute path?